### DISCUSSION TITLE
Servers Comunication

### DISCUSSION THREAD

**Discussion by ImHammer** [WaterdogPE/WaterdogPE#73] [Q&A]
Is there any way to make the proxy send and receive data to the server? So that the server can process this data, for example:

OtherServer -> [Player's Money] -> Proxy -> [Player's Money] -> Lobby

**Comment by Alemiz112** [comment]
You can found some useful info about proxy and downstream communication in our [docs](https://docs.waterdog.dev/books/plugins/page/proxy-communication).

**Reply by ImHammer** [reply]
Did not quite understand in docs...

**Reply by Alemiz112** [reply]
Can you point out details which you find difficult to understand?
